apps/ledger-live-desktop/src/renderer/screens/account/BalanceSummary.tsx

```diff
@@ -106,23 +111,35 @@ export default function AccountBalanceSummary({
       </Box>
 
       <Box px={5} ff="Inter" fontSize={4} color="palette.text.shade80" pt={5}>
-        <Chart
-          magnitude={chartMagnitude}
-          color={chartColor}
-          // TODO we need to make Date non optional in live-common
-          data={history}
-          height={200}
-          tickXScale={range}
-          valueKey={displayCountervalue ? "countervalue" : "value"}
-          renderTickY={
-            discreetMode
-              ? () => ""
-              : displayCountervalue
-              ? renderTickYCounterValue
-              : renderTickYCryptoValue
-          }
-          renderTooltip={renderTooltip}
-        />
+        {account.type === "TokenAccount" && account.token.id === "vechain/vtho" ? (
```

Comment:
we must not do such specific logic in the generic part of our app. instead, blockchain team will have to allow this space to be "overridable".